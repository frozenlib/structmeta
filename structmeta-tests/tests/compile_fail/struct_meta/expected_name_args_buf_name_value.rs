#[structmeta_tests::attr_name_args(value = 10)]
fn my_func() {}

fn main() {}
