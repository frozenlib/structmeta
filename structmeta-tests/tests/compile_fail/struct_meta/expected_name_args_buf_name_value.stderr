error: expected `value(...)`, found `value = ...`
 --> $DIR/expected_name_args_buf_name_value.rs:1:45
  |
1 | #[structmeta_tests::attr_required_name_args(value = 10)]
  |                                             ^^^^^
